spec/support/app_helpers.rb

```diff
@@ -87,7 +87,7 @@ def expect_page_to_be_affiliated_with_org(options)
   end
 
   def expect_change_note(change_note, options)
-    within(".subsection-collection") do
+    within(".section-content") do
```

Comment:
@leenagupte The tests were previously failing here because up until now there had only ever been one year used in the test data, which meant that there'd only ever be one `subsection-collection` class; by adding more test data across different years, it broke the Capybara `within` selector.

I've instead changed this to be the parent of all `subsection-collection`s, which is the `section-content` class. I'm not sure if it was previously scoped to a `subsection-collection` for any good reason, such as perhaps same-named links repeating across different years, but from the previous test data I don't believe that's the case.